src/controlplane/grouper/group.go

```diff
@@ -1,4 +1,4 @@
-package controlplane
+package grouper
```

Comment:
if possible I would try to move to tests _test, ignore this if too refactor is required

Reply:
not sure what you mean here